scripts/README.md

```diff
@@ -10,4 +10,14 @@ node substitute_article_urls.js $TARGET
 For example, to point all links to innersourcecommons.org:
 ```
 node substitute_article_urls.js isc
+```
+
+## generate_learning_path_markdown.js
+
+A node script to generate markdown files required for hosting Learning Path on innersourcecommons.org.
+
+### Usage:
+```
+npm ci
+node generate_learning_path_markdown.js
```

Comment:
Do we need to integrate this into some sort of GitHub Action?

Reply:
That would be the long term aim I think yes. 

In my opinion, we should have a GitHub Action generate the markdown and raise a Pull Request on https://github.com/InnerSourceCommons/innersourcecommons.org if it had changed.